gulpfile.js

```diff
@@ -38,13 +40,27 @@ gulp.task('git:hooks:install', shell.task([
 gulp.task('typings:install', shell.task('typings install'));
 
 // write constants for Env to src file
-const fs = require('fs');
-const dotenv = require('dotenv');
+function parseDotFile(path, mustExist) {
+  try { return dotenv.parse(fs.readFileSync(path, {encoding: 'utf8'})); }
+  catch (e) { if (e.code != 'ENOENT' || mustExist) throw e; }
+  return {};
+}
+function encodeDotValue(val) {
+  if (['true', 'false', 'null', 'undefined'].indexOf(val) > -1) return val;
+  if (isNaN(val) || val == '') return "'" + val + "'";
+  return val;
+}
 gulp.task('env:write', function(cb) {
-  var env = dotenv.parse(fs.readFileSync('.env', { encoding: 'utf8' }));
+  var defaults = parseDotFile('env-defaults', true);
```

Comment:
Not sure how I feel about this, since it seems it might be a surprise to folks that something other than the `.env*` file is being loaded and used.  I'm gonna leave it be, since I am imagining possible confusion rather than having any factual issue with this, but it is not how most .env stuff gets loaded in other standard libs and other projects.